app/models/content/content.ts

```diff
@@ -63,9 +61,11 @@ export const ContentModel = types
       return self.creator && self.creator.isFollowing
     },
     get isLoading() {
-      return !(self.hasCached || self.hasFetchedDetails) ||
+      return (
+        !(self.hasCached || self.hasFetchedDetails) ||
```

Comment:
Do you think `(!self.hasCached && (!self.hasFetchedDetails || self.isFetchingDetails)) || (self.creator && self.creator.isLoading)` would be clearer?